oscovida/oscovida.py

```diff
@@ -1715,41 +1730,39 @@ def overview(country: str, region: str = None, subregion: str = None,
     Returns: subplots, cases (pandas Series), deaths (pandas Series)
     """
     if data is None:
-        c, d = get_country_data(country, region=region, subregion=subregion)
+        _c, _d = get_country_data(country, region=region, subregion=subregion)
     else:
-        c, d = data
-        assert isinstance(c.index[0], pd.Timestamp), f"The index of 'cases' is not of type `Timestamp`, " \
+        _c, _d = data
+        assert isinstance(_c.index[0], pd.Timestamp), f"The index of 'cases' is not of type `Timestamp`, " \
                                                      f"try to use `index=pd.DatetimeIndex(dates)`"
-        assert isinstance(d.index[0], pd.Timestamp), f"The index of 'deaths' is not of type `Timestamp`, " \
+        assert isinstance(_d.index[0], pd.Timestamp), f"The index of 'deaths' is not of type `Timestamp`, " \
```

Comment:
May make more sense to raise a `TypeError` in these cases instead of just an Assertion error.